scout/server/templates/utils.html

```diff
@@ -141,7 +141,7 @@ <h5 class="modal-title">Edit comment</h5>
   <div class="card panel-default">
     <div class="panel-heading"><span class="fa fa-book"></span>&nbsp;Activity</div>
     <ul class="list-group" style="max-height:400px; overflow-y: scroll;">
-      {% for event in events %}
+      {% for event in events|sort(attribute="created_at", reverse=True) %}
```

Comment:
You could also sort on the cursor / db call. I'm guessing the db implementation is slightly faster. But since they are few and all get shown anyway I doubt the performance hit is in any way noticeable.

Reply:
"This is more of a comment than a question".. 😉